rest-model/src/main/java/com/gentics/mesh/core/rest/schema/impl/SchemaCreateRequest.java

```diff
@@ -26,7 +26,7 @@
 
 	@JsonProperty(required = false)
 	@JsonPropertyDescription("Flag which indicates whether nodes which use this schema store additional child nodes.")
-	private boolean container = false;
+	private Boolean container = false;
```

Comment:
Remove default value and set the default value in the schema create endpoint handler.